spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerMutationFactory.java

```diff
@@ -48,7 +49,7 @@
 	 * Optional, then all columns are used.
 	 * @return The mutation operation to perform the action.
 	 */
-	Mutation upsert(Object object, Optional<Set<String>> includeColumns);
+	List<Mutation> upsert(Object object, Optional<Set<String>> includeColumns);
 
 	/**
 	 * Updates a single object in Spanner. The columns' values corresponding to the
```

Comment:
Update javadoc to say it's for all children, too. and mention for objects without interleaved it's just 1